### PR TITLE
Adds a new contractor module, the SCORPION

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request

https://user-images.githubusercontent.com/41448081/169436569-2621cc5e-02ba-455a-8837-39874b860b1f.mp4

This PR adds the SCORPION hook module, available to contractors for 3 Rep. When used, it deploys a hook launcher in your hand that's tied to the suit, and will retract on-drop. The hook, when it hits a target, will pull it rapidly forward in addition to dealing 25 stamina damage.

## How This Contributes To The Skyrat Roleplay Experience
Jungle had this idea and i really wanted to add it. Gives contractors more melee viability against long-range targets

## Changelog

:cl:
add: New module for Contractors added, the SCORPION hook module, costs 3 Rep.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/contractor/code/items/hook.dm [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+/obj/projectile/hook/contractor
+	icon = 'modular_skyrat/modules/contractor/icons/hook_projectile.dmi'
+	damage = 0
+	stamina = 25
+	chain_icon = 'modular_skyrat/modules/contractor/icons/hook_projectile.dmi'
+
+/obj/item/gun/magic/hook/contractor/process_fire(atom/target, mob/living/user, message, params, zone_override, bonus_spread)
+	if(prob(1))
+		user.say("+GET OVER HERE!+")
```
Comment (Ghommie) [inline modular_skyrat/modules/contractor/code/items/hook.dm]:
```suggestion
		user.say("+GET OVER HERE!+", forced = "scorpion hook")
```

**Comment by Ghommie** [comment]
By the by, is this something that the /tg/station upstream wouldn't want? You could PR it there if they're ok with it.

**Comment by Zonespace27** [comment]
> By the by, is this something that the /tg/station upstream wouldn't want? You could PR it there if they're ok with it.

TG doesn't have contractors anymore

**Comment by Ghommie** [comment]
> TG doesn't have contractors anymore

Maaan, I'm living in the past.

**Comment by Zonespace27** [comment]
> > TG doesn't have contractors anymore
> 
> Maaan, I'm living in the past.

TG is lame in many respects

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**